portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACSegments.path

```diff
@@ -306,6 +311,11 @@
 	<td>//div[contains(@class,'date-picker')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>MODAL_CANCEL_BUTTON</td>
+	<td>//div[contains(@class,'modal-footer')]//button[contains(text(),'Cancel')]</td>
+	<td></td>
+</tr>
```

Comment:
Hey @calvinctvo, I think you don't need to close the modal as your test has no more steps after that moment

Reply:
Hey @ViniLopes20 , for this case, I don't think we need to close the modal. But I think keeping the path for future tests that could use the Cancel button would save us a hassle since there's no Cancel button has been created for the modal popup yet, and it could be mistaken by `CANCEL_SEGMENT` path. What do you think

Reply:
Ohhh yess, I think it could be good for us. I think these button actions we can use the `viewGenericButton` macro, but if you think we need something more specific to be clearer, I think we can keep this path.

I don't think we intend to have tests that use AC in a language other than English to verify the functionality, but in some projects this is necessary and maybe using Paths that use a word that can be changed in the translation, you can make the path not work in these cases. But since our case is different, I think we could keep it if you think it's necessary.